src/App/Header/NavDropdown.tsx

```diff
@@ -69,6 +75,13 @@ export default function NavDropdown({ links }: Props) {
                       </NavLink>
                     )
                   )}
+                  {user && user !== "loading" && (
```

Comment:
Suggestion: `user === "loading"` used to show a loading indicator, so that the user knows something (else) is being loaded in the menu.

Reply:
Fixed in [a8ab169](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2815/commits/a8ab1696c67929db9823e083ab938660acd1830f).